web/impact/impact/tests/test_analyze_judging_round_view.py

```diff
@@ -0,0 +1,182 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+import json
+from jsonschema import Draft4Validator
+
+from django.urls import reverse
+
+from accelerator.tests.factories import (
+    CriterionOptionSpecFactory,
+    JudgeRoundCommitmentFactory,
+)
+from accelerator.tests.contexts import AnalyzeJudgingContext
+from impact.tests.api_test_case import APITestCase
+from impact.tests.utils import assert_fields
+from impact.v1.views import AnalyzeJudgingRoundView
+
+
+class TestAnalyzeJudgingRoundView(APITestCase):
+    def test_get(self):
+        option = CriterionOptionSpecFactory()
+        judging_round_id = option.criterion.judging_round_id
+        with self.login(email=self.basic_user().email):
+            url = reverse(AnalyzeJudgingRoundView.view_name,
+                          args=[judging_round_id])
+            response = self.client.get(url)
+            assert len(response.data) == 1
+            first_result = response.data["results"][0]
+            assert first_result["criterion_option_spec_id"] == option.id
+
+    def test_options(self):
+        option = CriterionOptionSpecFactory()
+        judging_round_id = option.criterion.judging_round_id
+        with self.login(email=self.basic_user().email):
+            url = reverse(AnalyzeJudgingRoundView.view_name,
+                          args=[judging_round_id])
+            response = self.client.options(url)
+            assert response.status_code == 200
+            get_options = response.data["actions"]["GET"]["properties"]
+            results = response.data["actions"]["GET"]["properties"]["results"]
+            get_options = results["item"]["properties"]
+            assert_fields(AnalyzeJudgingRoundView.fields().keys(), get_options)
+
+    def test_options_against_get(self):
+        option = CriterionOptionSpecFactory()
+        judging_round_id = option.criterion.judging_round_id
+        with self.login(email=self.basic_user().email):
+            url = reverse(AnalyzeJudgingRoundView.view_name,
+                          args=[judging_round_id])
+
+            options_response = self.client.options(url)
+            get_response = self.client.get(url)
+
```

Comment:
this space intentionally left blank?

Reply:
Whitespace, the final frontier...
Cancelled.